docs/source/usage/workflows/synchrotronRequirements.py

```diff
@@ -0,0 +1,98 @@
+# /* Copyright 2014-2024 Filip Optolowicz
```

Comment:
should be 2024 only, the same for other new files.

Reply:
moved to 
/lib/python/synchrotronRadiationExtension/synchrotronRequirements.py